libs/sdk-ui-kit/src/SettingWidget/StylingSettingWidget/StylingSettingWidget.tsx

```diff
@@ -148,13 +155,21 @@ const StylingSettingWidgetCore = <T extends StylingPickerItemContent>(
                         <Button
                             className="gd-button-secondary"
                             onClick={handleCancel}
-                            disabled={isApplyButtonDisabled}
+                            disabled={
+                                disabledCancelButton !== undefined
+                                    ? disabledCancelButton
+                                    : isApplyButtonDisabled
+                            }
                             value={intl.formatMessage({ id: "cancel" })}
                         />
                         <Button
                             className="gd-button-action"
                             onClick={handleApply}
-                            disabled={isApplyButtonDisabled}
+                            disabled={
```

Comment:
Can we rewrite lines 168 ~ 172 as below, using a nullish-coalescing operator.

```
disabled = {disabledApplyButton ?? isApplyButtonDisabled}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing